dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -153,68 +149,104 @@ private Response postWithEndpointState(String endpointId, String url, MediaType
 		return response;
 	}
 
+    /**
+     * Resolves remote IP address from request.
+     * @param request {@link HttpServletRequest}
+     * @return a String representing the remote IP address (or hostname)
+     */
+    private static String resolveRemoteIp(@Context final HttpServletRequest request) {
```

Comment:
this is a static method, not sure if the context annotation make sense here

Reply:
Removed.